fw/http_stream_sched.c

```diff
@@ -476,6 +517,13 @@ tfw_h2_change_stream_dep(TfwStreamSched *sched, unsigned int stream_id,
 	BUG_ON(!stream);
 	old_parent = stream->sched.parent;
 	BUG_ON(!old_parent);
+	old_parent_id  = (old_parent != &sched->root ?
```

Comment:
We can use preprocessor conditionals here, I'm not sure that this line will be outoptimized.